content/docs/rkvst-basics/getting-access-tokens-using-app-registrations/index.md

```diff
@@ -143,18 +143,18 @@ You should see a response with details about the App Registration's `CLIENT_ID`
 ```
 
 {{< caution >}}
-**Caution:** You **must** take note of the `SECRET` at this point - it can **not** be viewed again later.
+**Caution:** You **must** take note of the `SECRET` at this point - it can **not** be viewed again later and you will have to generate a new one.
```

Comment:
As above:
Can we state you would need a new one in the event of losing this one?

Reply:
Yep, added!